main.go

```diff
@@ -345,7 +361,15 @@ func main() {
 
 		return
 	}
-
+	/*
+		if *demoUptime != "" && !SystemConfig.DontPurgeUptimeData{
+			log.Info("BUILDING UPTIME DEMO DATA AND EXITING...")
+			log.Warning("Starting from date: ", time.Now().AddDate(0, 0, -30))
+			demo.DemoInit(*demoUptime, *demoApiMode, *demoApiVersionMode)
+			demo.GenerateDemoUptimeData(time.Now().AddDate(0, 0, -30), 30, *demoUptime, UptimePump.WriteUptimeData)
+
+			return
+		}*/
```

Comment:
I think we can remove this if it's not in use, at least you always have git history to revert it back :)

Reply:
@vaske good catch! I was trying something there and forgot to delete it 👍